gravitee-node-license/src/main/java/io/gravitee/node/license/DefaultLicenseManager.java

```diff
@@ -0,0 +1,76 @@
+package io.gravitee.node.license;
+
+import io.gravitee.node.api.license.*;
+import io.gravitee.plugin.core.api.PluginRegistry;
+import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author Jeoffrey HAEYAERT (jeoffrey.haeyaert at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+@Slf4j
+public class DefaultLicenseManager implements LicenseManager {
+
+    public static final OSSLicense OSS_LICENSE = new OSSLicense();
+
+    private final Map<String, Optional<License>> organizationLicenses = new ConcurrentHashMap<>();
+    private License platformLicense;
+    private final PluginRegistry pluginRegistry;
+
+    public DefaultLicenseManager(PluginRegistry pluginRegistry) {
+        this.pluginRegistry = pluginRegistry;
+    }
+
+    @Override
+    public void registerOrganizationLicense(@NonNull String organizationId, License license) {
+        this.organizationLicenses.put(organizationId, Optional.ofNullable(license));
+    }
+
+    @Override
+    public void registerPlatformLicense(License license) {
+        this.platformLicense = license;
+    }
+
+    @Override
+    public License getOrganizationLicense(String organizationId) {
+        return organizationLicenses.getOrDefault(organizationId, Optional.empty()).orElse(null);
```

Comment:
see my other comments, you could return an Optional directly...

Reply:
Same here